src/manager/tooltipManager.js

```diff
@@ -50,15 +67,15 @@ var TooltipManager = class TooltipManager extends MsManager {
     }
 
     createTooltip(actor, params) {
-        if (!params.text) {
-            if (actor instanceof St.Label || actor instanceof Clutter.Text) {
-                const clutterText =
-                    actor instanceof St.Label
-                        ? actor.get_clutter_text()
-                        : actor;
-                if (clutterText.get_layout().is_ellipsized()) {
-                    params.text = clutterText.get_text();
-                }
+        // Always get the up-d-ate value
+        params.text = undefined;
```

Comment:
The params.text is supposed to allow us to set tooltip for other actor that are not St.Label or Clutter.Text like icon button you can't assign it to undefined...